aiida_common_workflows/workflows/relax/nwchem/generator.py

```diff
@@ -164,11 +164,34 @@ def get_builder(
         if magnetization_per_site:
             raise ValueError('magnetization per site not yet supported')
 
+        # Add a unit cell to the geometry stanza
+        add_cell = orm.Bool(True)
+
         # Special case of a molecule in "open boundary conditions"
         if structure.pbc == (False, False, False):
             warnings.warn('PBCs set to false in input structure: assuming this is a molecular calculation')
-            parameters['nwpw']['monkhorst-pack'] = '1 1 1'  # Gamma only
-            parameters['nwpw']['cutoff'] = 250
+            add_cell = orm.Bool(False)
+            # We don't use the geometry stanza for the cell, but add it in the parameters
+            parameters['nwpw']['simulation_cell angstroms'] = {
+                'lattice': {
+                    'lat_a': structure.cell_lengths[0],
+                    'lat_b': structure.cell_lengths[1],
+                    'lat_c': structure.cell_lengths[2],
+                    'alpha': structure.cell_angles[0],
+                    'beta': structure.cell_angles[1],
+                    'gamma': structure.cell_angles[2],
+                }
+            }
+            parameters['driver']['redoautoz'] = ''  # To ensure internal coordinates are refreshed
+            parameters['nwpw']['cutoff'] = 140
+            parameters['task'] = 'pspw optimize'
+
+            _ = parameters['nwpw'].pop('monkhorst-pack')
+            _ = parameters['nwpw'].pop('ewald_rcut')
+            _ = parameters['nwpw'].pop('ewald_ncut')
+            _ = parameters['nwpw'].pop('smear')
+            _ = parameters['nwpw'].pop('scf')
+            _ = parameters['nwpw'].pop('loop')
```

Comment:
Don't need to assign the popped value if you don't use it and maybe it's best to add `None` as the second argument so it won't raise if the key doesn't exist.
```suggestion
            parameters['nwpw'].pop('monkhorst-pack', None)
            parameters['nwpw'].pop('ewald_rcut', None)
            parameters['nwpw'].pop('ewald_ncut', None)
            parameters['nwpw'].pop('smear', None)
            parameters['nwpw'].pop('scf', None)
            parameters['nwpw'].pop('loop', None)
```

Reply:
Done, thanks!